src/main/java/ToDoList.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+
+public class ToDoList {
+    private static final ArrayList<Task> TaskList = new ArrayList<>(10);
+    private static int NumTasks = 0;
+    public static void addItem (String in) {
+        Task task = new Task(in);
+
+        TaskList.add(task);
+        NumTasks += 1;
+        System.out.println("added: " + in);
+    }
+    public static int getNumItems() {
+        return NumTasks;
+    }
+    public static void viewList () {
+        for (int i = 0; i < TaskList.size(); ++i) {
+            System.out.print(i+1 + ". ");
+
+            System.out.println(TaskList.get(i).getTask());
+        }
+    }
+    public static void markDone (int index) {
+        if (TaskList.get(index).getStatusIcon().equals(" ")) {
```

Comment:
Is this if statement necessary? I think markDone would change the status icon to "X" independent of what is originally is